src/components/ProfileResetPassword/ProfileResetPassword.tsx

```diff
@@ -0,0 +1,337 @@
+import React, { useState } from "react";
+import styles from "./styles";
+import { Modal, View, Pressable, TextInput, Text, Linking } from "react-native";
+import SvgIcon from "../../../assets/SvgIcon";
+import globalStyles from "../../globalStyles";
+import emailjs, { init } from "@emailjs/browser";
+
+export const ProfileResetPassword = ({ visible, setVisible, user }: any) => {
+  const [email, onChangeEmail] = useState("");
+  const [code, onChangeCode] = useState("");
+  const [newPW, onChangeNewPW] = useState("");
+  const [confirmPW, onChangeConfirmPW] = useState("");
+  const [edited, setEdited] = useState(false);
+  const [currScreen, setCurrScreen] = useState("reset");
+  const [currPin, setCurrPin] = useState(0);
+
+  const emailRegex = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/;
+
+  const reset = () => {
+    onChangeEmail("");
+    onChangeCode("");
+    onChangeNewPW("");
+    onChangeConfirmPW("");
+    setCurrScreen("reset");
+    setEdited(false);
+    setVisible(false);
+  };
+
+  init("bpUmfdhrALYzPBWpx"); //initializes emailJS userID (only 200 emails a month...)
```

Comment:
complete oversight on my part, but this was something we also did in the web repo. Make sure to store this valuable user credential/secret in the .env file and reference it that way. We don't want to publicly expose API keys/credentials like this on repos in general since other people could maliciously use it

Reply:
should i create a constants folder and do it like how albert had the vars saved in web?

Reply:
You should put the service id and user id in the .env file, but you can put the template id in a constants file